src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -69,7 +69,9 @@ public function copy($originFile, $targetFile, $overwriteNewerFiles = false)
             }
 
             // Like `cp`, preserve executable permission bits
-            @chmod($targetFile, fileperms($targetFile) | (fileperms($originFile) & 0111));
+            if (stream_is_local($originFile)) {
```

Comment:
I was reading about the `stream_is_local()` function and I saw this user comment: http://php.net/manual/en/function.stream-is-local.php#121126 Should we take that into account?